src/main/steps/prl-cases/pin-activation/case-activated/content.ts

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-non-null-asserted-optional-chain */
+import { TranslationFn } from '../../../../app/controller/GetController';
+import { FormContent, GenerateDynamicFormFields } from '../../../../app/form/Form';
+import { interpolate } from '../../../../steps/common/string-parser';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const en = () => ({
+  caption: 'Case number #{caseNumber}',
+  title: 'Case added to your account',
+  text: 'The case can now be seen on your child arrangements and family injunction account.',
+  continue: 'Continue',
+});
+
+export const cy = () => ({
+  caption: 'Case number -welsh #{caseNumber}',
+  title: 'Case added to your account -welsh',
+  text: 'The case can now be seen on your child arrangements and family injunction account. -welsh',
+  continue: 'Continue -welsh',
+});
+
+const languages = {
+  en,
+  cy,
+};
+
+let updatedForm: FormContent;
+
+const updateFormFields = (form: FormContent, formFields: FormContent['fields']): FormContent => {
+  updatedForm = {
+    ...form,
+    fields: {
+      ...formFields,
+      ...(form.fields ?? {}),
+    },
+  };
+
+  return updatedForm;
+};
+
+export const generateFormFields = (): GenerateDynamicFormFields => {
+  const errors = {
+    en: {},
+    cy: {},
+  };
+
+  const fields = {};
+
+  return { fields, errors };
+};
+
+export const form: FormContent = {
+  fields: {},
+  submit: {
+    text: l => l.continue,
+  },
+};
+
+export const getFormFields = (): FormContent => {
+  return updateFormFields(form, generateFormFields().fields);
+};
+
+export const generateContent: TranslationFn = content => {
+  const caseNumber = content.userCase?.id! ?? undefined;
```

Comment:
@giorgi-hmcts userCase could be null when you are in this screen. Please recheck & correct the ref